src/calculateRentalCost.js

```diff
@@ -3,8 +3,25 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentCost = 40;
+
+  const longTerm = 7;
+  const shortTerm = 3;
+
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+
+  if (days >= longTerm) {
+    return days * dayRentCost - longTermDiscount;
+  }
+
+  if (days >= shortTerm) {
+    return days * dayRentCost - shortTermDiscount;
```

Comment:
This 'if' statement also correctly uses a return, which is good as it keeps the code simple and clear.